src/components/Zoo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { shallowEqual, useSelector } from 'react-redux';
+import Animal from './Animal';
+import Landing from './landing';
+
+const Zoo = () => {
```

Comment:
- [ ] On your homepage, kindly ensure you implement the filtering mechanism (like a search box) as outlined in the requirements.

  ![furkan-home](https://user-images.githubusercontent.com/36542986/164279679-888ccfe5-7a9d-4dd9-b303-f9f05efcbd35.jpg)

Reply:
Hello @TedLivist I have effected this requested additional feature